src/components/SlideButton/SlideButton.vue

```diff
@@ -0,0 +1,35 @@
+<template>
+  <div class="wrapper">
+    <div class="slider-wrapper">
+      <div
+        ref="slider"
+        :class="`slider-track ${sliderClass}`"
+      >
+        <div
```

Comment:
I know we used to format elements this way (with the first attribute broken out onto the following line) however we have a new approach with Vue (please see other `.vue` files in this project for examples):

```js
<div ref="overlay"
     class="overlay"
     :style="[overlayStyle]"
/>
```

Please update the rest of the markup in this file accordingly, thanks!